scripts/convertMedia.ts

```diff
@@ -2,55 +2,94 @@ import { CopyOptions, cpSync } from 'fs';
 import path from 'path';
 import { Task, TaskOutput } from './Task';
 
-function cpSyncOptional(source: string, destination: string, opts?: CopyOptions): void {
+function cpSyncOptional(source: string, destination: string, opts?: CopyOptions): boolean {
     try {
         cpSync(source, destination, opts);
+        return true;
     } catch (e) {
         // source doesn't exist, ok.
+        return false;
     }
 }
-
-export function convertMedia(dataDir: string) {
+/**
+ * Copies styles, fonts, images, illustrations, audio, and timings from supplied data folder to static.
+ */
+export function convertMedia(dataDir: string, verbose: boolean) {
     // Copy the css stylesheets
     cpSync(path.join(dataDir, 'styles'), path.join('static', 'styles'), { recursive: true });
+    if (verbose)
+        console.log(`copied ${path.join(dataDir, 'styles')} to ${path.join('static', 'styles')}`);
 
     // Copy the font files
     cpSync(path.join(dataDir, 'fonts'), path.join('static', 'fonts'), { recursive: true });
+    if (verbose)
+        console.log(`copied ${path.join(dataDir, 'fonts')} to ${path.join('static', 'fonts')}`);
 
     // Copy the images
-    cpSyncOptional(path.join(dataDir, 'images'), path.join('static', 'images'), {
-        recursive: true
-    });
+    if (
+        cpSyncOptional(path.join(dataDir, 'images'), path.join('static', 'images'), {
+            recursive: true
+        })
+    ) {
```

Comment:
This might be a little cleaner of the verbose flag is passed into the cpSyncOptional and handled there. Or separate copyRequired and copyOptional functions with the verbose flag so that there is not so much duplication.

Actually, don't worry about this. I can clean it up later...